### PR TITLE
Basic UI for starting a job

### PR DESCRIPTION
**Pull request by yoninachmany**
r @anoshaminai @sandlerben 
Create a basic web form that takes in a URL (to the audio link) and email addresses, starts a job, and goes back to form

screen shot 2016-03-26 at 1 09 52 pm
screen shot 2016-03-26 at 2 11 00 pm

### CODE REVIEW

File: web/routes.go [hunk 1]
```diff
@@ -55,21 +61,35 @@ func helloHandler(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprintf(w, "Hello %s!", args["name"])
 }
 
-// initiateTranscriptionJobHandle takes a POST request containing a json object,
-// decodes it into an audioData struct, and returns appropriate message.
-func initiateTranscriptionJobHandler(w http.ResponseWriter, r *http.Request) {
-	var jsonData transcriptionJobData
+// initiateTranscriptionJobHandlerJSON takes a POST request containing a json object,
```
Comment (sandlerben) [inline web/routes.go:61]: Why is this commented out?

File: web/routes.go [hunk 2]
```diff
@@ -55,21 +61,35 @@ func helloHandler(w http.ResponseWriter, r *http.Request) {
...
+// decodes it into a transcriptionJobData struct, and starts a transcription task.
+func initiateTranscriptionJobHandler(w http.ResponseWriter, r *http.Request) {
+	formData := transcriptionJobData{
+		AudioURL:       r.FormValue("AudioURL"),
+		EmailAddresses: r.Form["EmailAddresses"],
 	}
-
 	executer := tasks.DefaultTaskExecuter
-	id := executer.QueueTask(transcription.MakeTaskFunction(jsonData.AudioURL, jsonData.EmailAddresses))
+	id := executer.QueueTask(transcription.MakeTaskFunction(formData.AudioURL, formData.EmailAddresses))
```
Comment (sandlerben) [inline web/routes.go]: Why do you need to even put the data in a struct? Couldn't you just make the `r.FormXXX` calls here?

File: web/routes.go [hunk 3]
```diff
@@ -85,3 +105,24 @@ func jobStatusHandler(w http.ResponseWriter, r *http.Request) {
 	status := executer.GetTaskStatus(id)
 	w.Write([]byte(status.String()))
 }
+
+func formHandler(w http.ResponseWriter, r *http.Request) {
+	const tpl = `
```
Comment (sandlerben) [inline web/routes.go]: Please put this in an actual html file using whatever conventions Go has for static files.

File: web/routes.go [hunk 4]
```diff
@@ -102,22 +101,6 @@ func jobStatusHandler(w http.ResponseWriter, r *http.Request) {
...
-	    <div>URL:<input type="url" name="AudioURL" required></div>
-			<div>Email Addresses:<input type="email" name="EmailAddresses" multiple required></div>
-			<div><input type="submit" value="Submit"></div>
-	  </form>
-	</form>
-	</body>
-	</html>
-	`
-	t, _ := template.New("webpage").Parse(tpl)
+	t, _ := template.ParseFiles("web/form.html")
```
Comment (sandlerben) [inline web/routes.go]: Never ignore errors!

Comment (sandlerben) [inline web/routes.go]: Did the linter not complain about this?

**Comment by sandlerben** [comment]
Left a small comment, besides that lgtm

File: web/routes.go (hunk 4 above)
Comment (yoninachmany) [inline web/routes.go]: Linter did not complain -- still, I should handle error, right?

Comment (sandlerben) [inline web/routes.go]: always

**Comment by sandlerben** [comment]
Let me know when you make the changes we talked about today. I'll give it a review then.

**Pull request merged**